pkgs/top-level/aliases.nix

```diff
@@ -81,6 +83,8 @@ mapAliases ({
   atom-beta = throw "'atom-beta' has been removed because discontinued and deprecated. Consider using 'pulsar', a maintained fork"; # Added 2023-10-01
   atomEnv = throw "'atomEnv' has been removed because 'atom' is discontinued and deprecated. Consider using 'pulsar', a maintained fork"; # Added 2023-10-01
   atomPackages = throw "'atomPackages' has been removed because 'atom' is discontinued and deprecated. Consider using 'pulsar', a maintained fork"; # Added 2023-10-01
+  # Post 24.11 branch-off, this should throw an error in aliases.nix.
+  autoAddOpenGLRunpathHook = lib.warn "autoAddOpenGLRunpathHook has been deprecated in favor of autoAddDriverRunpathHook" autoAddDriverRunpathHook;
```

Comment:
Unfortunately, we can't just `inherit` the `lib.warn` in `cudaPackages`, because Ofborg recurses into it.

I suggest that we just spare a warning for `autoAddOpenGLRunpathHook` (it's not used that widely anyway) and wait for https://github.com/NixOS/rfcs/blob/master/rfcs/0127-issues-warnings.md

Reply:
Perhaps I should just remove the warnings in general? IIRC stuff usually gets added to `aliases.nix` and after some time they just throw an error.

Reply:
I think displaying warnings suggesting the new alternatives is not a bad thing. At least so long as they don't create much noise, and as long as the CI (Ofborg, Hydra) is happy.

Both are satisfied (I think) in the case of `addOpenGLRunpath`